Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2033,6 +2033,7 @@ extension BrowserViewController {
             guard self.onboardingUserResearch?.updatedLPVariables != nil else {
                 return
             }
+            Sentry.shared.send(message: "Failed to fetch A/B test variables from LP")
```

Comment:
@garvankeeley Added a simple message for sentry if we are unable to fetch A/B test variables from LP